orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/TerminateInstancesTask.groovy

```diff
@@ -38,13 +40,23 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
   @Autowired
   TerminatingInstanceSupport instanceSupport
 
+  @Autowired
+  TrafficGuard trafficGuard
+
   @Override
   TaskResult execute(Stage stage) {
     String cloudProvider = getCloudProvider(stage)
     String account = getCredentials(stage)
 
     List<TerminatingInstance> remainingInstances = instanceSupport.remainingInstances(stage)
 
+    trafficGuard.verifyInstanceTermination(
+      stage.context.instanceIds as List<String>,
+      account,
+      Location.region(stage.context.region as String),
```

Comment:
For this case, it calls out to Clouddriver's `AbstractRegionAsgInstanceIdsDescription`, which takes a single region.